release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -314,6 +314,20 @@ v1.13.
 scaffolded to use Go modules. The `operator-sdk new` command's `--dep-manager`
 flag has likewise been removed.
 
+[discrete]
+[id="ocp-4-3-cluster-logging-changes"]
+==== Cluster logging *out_forward* configuration changes
+
+Changes introduced by the new log forward feature modified the support for *out_forward* starting with the {product-title} 4.3 release. In {product-title} 4.3, you create a ConfigMap, as described below, to configure out_forward. Any updates to the `secure-forward.conf` section of the Fluentd ConfigMap are removed. You can copy your current `secure-forward.conf` section before upgrading to use when creating the `secure-forward` ConfigMap. 
```

Comment:
This is a little confusing.  We need to provide some sort of clarification distinguishing the `secure-forward.conf` section of the 4.2 and prior CM and  the new configmap.  It just needs to be more obvious and explicit

Reply:
8020ee4d493205d36603b79f775a289f43994a79